### PR TITLE
[FluidDynamicsApplication] Two-fluid inlet: Some corrections

### PR DESCRIPTION
**Pull request by swenczowski**
During tests on application, a bug in the implementation of the two-fluid inlet surfaced. The following correction is suggested and improvements following the hints in preceding PR (@philbucher) were made.

Sorry for the inconvenience of a second PR but the error was not visible in the test cases.

### CODE REVIEW

File: applications/FluidDynamicsApplication/custom_processes/two_fluids_inlet_process.cpp [hunk 1]
```diff
@@ -59,7 +59,7 @@ TwoFluidsInletProcess::TwoFluidsInletProcess(
 
     // normalization of itnerface normal vector
     if ( norm_2( mInterfaceNormal ) > 1.0e-7 ){
-        mInterfaceNormal = mInterfacePoint / norm_2( mInterfaceNormal );
+        mInterfaceNormal = mInterfaceNormal / norm_2( mInterfaceNormal );
```
Comment (rubenzorrilla) [inline applications/FluidDynamicsApplication/custom_processes/two_fluids_inlet_process.cpp]:
```suggestion
        mInterfaceNormal /= norm_2( mInterfaceNormal );
```

**Review by rubenzorrilla — DISMISSED** [review]

**Comment by jcotela** [comment]
@swenczowski once the tests pass, I would suggest adding them to the list of registered tests in `test_FluidDynamicsApplication.py`, to make sure that we are not inadvertently breaking them again in the future. If they are small enough that they run in about one second, I'd ask you to add them to the nightly suite.

**Review by rubenzorrilla — APPROVED** [review]

**Comment by loumalouomega** [comment]
@swenczowski do you still need the branch?

**Comment by swenczowski** [comment]
Thank you!

**Pull request merged**